### ISSUE TITLE
Improve UI for tempo scaling

### ISSUE THREAD

**Issue by oMrSmith** [musescore/MuseScore#17595]
### Your idea

My proposal:
- provide direct access for tempo scaling
- make the tempo scaling lever more compact

Possible UI:

![image](https://github.com/musescore/MuseScore/assets/78870770/f3c6e0fa-a851-44b7-9c3f-d4c8e155b804)
  

### Problems to be solved

- tempo scaling is currently hidden
- tempo scaling uses a lever, which is much bigger than it would need to be
- accessing tempo scaling is laborious as it requires 2 extra clicks to undock and then redock 
- in MU3 you could at least use a shortcut (F11) to access tempo scaling 

### Additional context

Ideally the playback toolbar should be customizable.

**Comment by oMrSmith** [comment]
Now in MS 4.5 tempo scaling is way more accessible - small change, big difference.
The only disadvantage I see, is that the tempo indicator does not indicate, 
whether the tempo has been scaled or not - unless you click on it,
but this is not a big deal.

**Comment by MarcSabatella** [comment]
Good call, though, on the lack of a visible indicator.  Might be nice to have a separate issue to request that.
